docs/tutorials/e2e/boost/consumeData.md

```diff
@@ -42,51 +42,76 @@ curl --location 'http://localhost/alice/management/v2/contractnegotiations' \
 --header 'X-Api-Key: password' \
 --data-raw '{
  "@context": {
-  "odrl": "http://www.w3.org/ns/odrl/2/"
+    "odrl": "http://www.w3.org/ns/odrl/2/"
  },
  "@type": "NegotiationInitiateRequestDto",
- "connectorAddress": "http://alice-controlplane:8084/api/v1/dsp",
+ "connectorAddress": "http://bob-controlplane:8084/api/v1/dsp",
  "protocol": "dataspace-protocol-http",
- "connectorId": "BPNL000000000001",
- "providerId": "BPNL000000000001",
+ "providerId": "BPNL000000000002",
  "offer": {
-  "offerId": "MQ==:MQ==:ZDM4Nzk3NmUtZjA0Ny00ZmNjLWFhNWItYjQwYmVkMDBhZGYy",
-  "assetId": "3",
-  "policy": {
-   "@type": "odrl:Set",
-   "odrl:permission": {
-    "odrl:target": "3",
-    "odrl:action": {
-     "odrl:type": "USE"
-    },
-    "odrl:constraint": {
-     "odrl:or": {
-      "odrl:leftOperand": "BusinessPartnerNumber",
-      "odrl:operator": {
-                            "@id": "odrl:eq"
-                        },
-      "odrl:rightOperand": "BPNL000000000002"
-     }
+    "offerId": "Mw==:Mw==:ODRkYjJhZjQtZjMxOC00ZjgyLThjMjktODQwZThjYjBjNjVl",
+    "assetId": "3",
+    "policy": {
+        "@type": "odrl:Set",
+        "odrl:permission": [],
+        "odrl:prohibition": [],
+        "odrl:obligation": [],
+        "odrl:target": "3"
     }
-   },
-   "odrl:prohibition": [],
-   "odrl:obligation": [],
-   "odrl:target": "3"
-  }
  }
-}'
+}' | jq
+```
+
+In the response, Alice gets a UUID. This is the ID of the created contract negotiation. Alice can now use this ID to see the current status of the negotiation and - if the negotiation was successful - the ID of the created contract agreement.
```

Comment:
Could we name the field of the UUID?

Reply:
Why not, of course, if it helps the user to understand the whole concept better!